examples/AzureML-REST-API/compute.md

```diff
@@ -0,0 +1,170 @@
+client_secret = "<my-clent-secret>"
+user_nmae = "<my-user-name>"
+
+subid = "<my-subscription-id>"
+rg = "<my-workspace-resource-group>"
+
+auth_context = AuthenticationContext("https://login.microsoftonline.com/{}.onmicrosoft.com".format(user_name))
+
+resp = auth_context.acquire_token_with_client_credentials("https://management.azure.com/",client_id,client_secret)
+
+token = resp["accessToken"]
+```
+
+## Create AML Compute
+
+ - Need "token" from Authenticate against Azure
+ - Need a existed AML workspace
+ - Need a existed Azure Virtual Network
+ 
+ 1. Set the basic config
+ ``` python
+# general config
+subid = "<my-subscription-id>"
+rg = "<my-workspace-resource-group>"
+ws = "<my-workspace-name>"
+api_version = "2019-06-01"
+```
+
+2. set the compute resource config.
+```python
+# location, ex: westus2, eastus
+location = "<your workspace location>" 
+
+ # compute target type ex: VirtualMachine  AmlCompute
+compute_type = "<compute type>"
+
+ # ex: Standard_D1, Standard_F64s_v2
+vmSize = "<VmType>"
+
+# ssh account and password for compute
+admin_user_name= "<admin>" 
+admin_user_passwd= "<nimda123@S>" 
+
+# Dedicated or Low Priority
+vmPriority = "<Dedicated>" 
+
+# set basic node schedule policy
+maxNodeCount = <maxNumber>
+minNodeCount = <lowNumner>
+
+# set node idle time tolerance, time format according to ISO8601
+# 1 hour 30 minutes --> PT1H30M
+# 5 minutes --> PT5M
+# 2 hours --> PT2H
+nodeIdleTimeBeforeScaleDown = "PT5M" #  set 5 minutes tolerance
+
+# virtual network id you have created, 
+# get more info: https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-overview
+network_id = "<virtual network id>"
+
+# subnet id of virtual network
+subnet_id =  "<default>"
+```
+3. send request
+```
+body = {
+          "location": location, 
+          "properties": {
+            "computeType": compute_type,
+            "properties": {
+              "vmSize": vmSize,
+              "vmPriority": vmPriority,
+              "scaleSettings": {
+                "maxNodeCount": maxNodeCount,
+                "minNodeCount": minNodeCount,
+                "nodeIdleTimeBeforeScaleDown": nodeIdleTimeBeforeScaleDown
+              }
+            }
+          },
+          "userAccountCredentials":{
+            "adminUserName": admin_user_name,
+            "adminUserPassword": admin_user_passwd
+          },
+          "subnet":{
+            "id":"/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Network/virtualNetworks/{}/subnets/{}".format(subid, rg, network_id, subnet_id)
+         }
+        }
+
+header = {'Authorization': 'Bearer ' + token, "Content-type": "application/json"}
+create_compute_url = "https://management.azure.com/subscriptions/{}/resourceGroups/{}/providers/Microsoft.MachineLearningServices/workspaces/{}/computes/{}?api-version={}".format(subid, rg, ws, compute_name, api_verison)
+
+resp = requests.put(create_compute_url, headers=header, json=body)
+print resp.text
+```
+
```

Comment:
Adding an example for 
1. how to poll for status
2. update compute 

will be more helpful

Reply:
Rest operation of resource seems don't have a progress to poll. Is "get method" enough?

Reply:
And update compute has the same interface with create compute.

Reply:
including this information in the sample can be helpful -  "compute update" interface is same with "compute create" interface.

Reply:
Sure, make sense. Will add this information.